stripper/ze_mapeadores_museum_v1.cfg

```diff
@@ -548,3 +548,166 @@ modify:
 		"OnUser2" "!activator,SetLocalOrigin,9590 11906 14023,10,1"
 	}
 }
+
+// Fix evading the spawn teleport.
+modify:
+{
+	match:
+	{
+		"classname" "trigger_teleport"
+		"targetname" "MuseumSpawnMainTeleport"
+	}
+	replace:
+	{
+		"model" "*544"
+		"origin" "-14336 14336 13952"
+	}
+}
+
+// Kill the buttons in spawn when pressed.
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"targetname" "secretbutton"
+	}
+	delete:
+	{
+		"OnPressed" "!selfKill0.11"
+	}
+	insert:
+	{
+		"OnPressed" "!self,Kill,,,1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"targetname" "secretpotato"
+	}
+	insert:
+	{
+		"OnPressed" "!self,Kill,,,1"
+	}
+}
+
+// Fix "func_breakable"s set to "None" material type producing wooden gibs.
+modify:
+{
+	match:
+	{
+		"classname" "func_breakable"
+		"material" "10"
+	}
+	replace:
+	{
+		"PerformanceMode" "1"
+	}
+}
+
+// Kill the ending button on "SSS-Lite" when it's pressed.
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"targetname" "hich_sss_endbutton"
+	}
+	insert:
+	{
+		"OnPressed" "!self,Kill,,,1"
+	}
+}
+
+// Fix "SSS-Lite" ending teleport using incorrect name for its filter.
+modify:
+{
+	match:
+	{
+		"classname" "trigger_teleport"
+		"targetname" "hich_sss_endtp_ct"
+	}
+	replace:
+	{
+		"filtername" "hich_filter_allow_ct"
+	}
+}
+
+// Fix "GOLEVATOR" elevator getting people stuck on spawn and fix delaying.
+add:
+{
+	"classname" "trigger_once"
+	"targetname" "hich_gol_antistuck1"
+	"model" "*298"
+	"spawnflags" "1"
+	"origin" "12288 0 -12208"
+	"OnStartTouch" "!activatorRunScriptCodeflOrigin <- self.GetOrigin(); self.SetOrigin(Vector(flOrigin.x, flOrigin.y, -12192)); delete flOrigin;-1"
```

Comment:
This is my first time seeing `delete` in Squirrel. I looked at Squirrel documentation and it seems like it's used for deleting table slots, any reason you're calling it directly on a Vector?

Reply:
I am temporarely storing player origin in their script scope then deleting it after using it, delete is used to delete a variable since in this case it isn't a local variable and won't be auto deleted, it's a personal choice I could also do
self.SetOrigin(Vector(self.GetOrigin().x, self.GetOrigin().y, -12192)); but I don't see point in getting the same variable twice.

Reply:
Well the point here would more be that deleting variables isn't needed in the first and is never really done, but as long as it works I don't see any harm.